content/en/docs/tasks/observability/distributed-tracing/overview/index.md

```diff
@@ -90,32 +90,3 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
-## Trace sampling
-
-Istio captures a trace for all requests by default when installing with the demo profile.
-For example, when using the Bookinfo sample application above, every time you access
-`/productpage` you see a corresponding trace in the
-dashboard. This sampling rate is suitable for a test or low traffic
-mesh. For a high traffic mesh you can lower the trace sampling
-percentage in one of two ways:
-
-* During the mesh setup, use the option `values.pilot.traceSampling` to
```

Comment:
There are other documents that mention this setting. Should I remove them or link to the experimental doc? Examples include setting up tracing for jaeger, zipkin and stackdriver.

Reply:
Guidance @frankbu @ericvn

Reply:
Add a link to experimental docs.